filter/etc/e-smith/templates/etc/rspamd/rspamd.conf/60WorkerProxy

```diff
@@ -17,7 +17,6 @@ worker "rspamd_proxy" \{
 
     # number of worker
     count = {$rspamd{MaxProcesses} || '1'}
-    type = "proxy";
```

Comment:
Just out of Curiosity Why this line has not the corresponding Worker section?

Reply:
I really do not know :) 
https://github.com/vstakhov/rspamd/blob/master/conf/worker-proxy.inc

Reply:
The suggested link still has type=proxy

https://github.com/vstakhov/rspamd/blob/0d217fc8880ed3b6902f72022da7d38f324c368a/conf/worker-proxy.inc#L3

Reply:
yes, in a meanwhile you have this also : [Minor] Remove unnecessary type https://github.com/vstakhov/rspamd/commit/38dfe192fc706ffcad31296105dca7248cca6104

did the main developer missed it :-?

Reply:
it seems also that with 1.7.4 we have more options in the worker-proxy, see https://github.com/vstakhov/rspamd/commit/90f15439af708a6c36a10fa8636f552c9d6b999a

we could add 

- max_retries= 5;
- spam_header = "X-Spam-Flag";

Reply:
question asked for type=proxy on telegram

Reply:
answer of the lead dev , it is a mistake, we could remove `type=proxy`